client/ayon_deadline/lib.py

```diff
@@ -8,3 +10,33 @@
     "karma_rop", "vray_rop", "redshift_rop",
     "renderFarm", "usdrender", "publish.hou"
 ]
+
+
+def get_ayon_render_job_envs() -> "dict[str, str]":
+    """Get required env vars for valid render job submission."""
+    return {
+        "AYON_LOG_NO_COLORS": "1",
+        "AYON_RENDER_JOB": "1",
+        "AYON_BUNDLE_NAME": os.environ["AYON_BUNDLE_NAME"]
```

Comment:
I would say that `AYON_LOG_NO_COLORS` and `AYON_BUNDLE_NAME` should be defined by ayon-core in future.

Reply:
I'd argue that MOST of these are actually - but the first step is having this implemented - released, and then at some point get future addons to implement the feature.

Reply:
I would argue that deadline should set up only `AYON_RENDER_JOB`, but can be changed in future.

Reply:
EDITED: Maybe we should add function that would set environment variable based on job type rather... Can be done in future PRs.